mmv1/products/tpuv2/Vm.yaml

```diff
@@ -0,0 +1,72 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'Vm'
+description: |
+  A Cloud TPU VM instance.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Official Documentation': 'https://cloud.google.com/tpu/docs/'
+  api: 'https://cloud.google.com/tpu/docs/reference/rest/v2/projects.locations.nodes'
+base_url: projects/{{project}}/locations/{{zone}}/nodes
+self_link: projects/{{project}}/locations/{{zone}}/nodes/{{name}}
+create_url: projects/{{project}}/locations/{{zone}}/nodes?nodeId={{name}}
```

Comment:
Where is the release track configured for this endpoint, ie we want to hit v2alpha1 proto

Reply:
Done, I updated it in `mmv1/products/tpuv2/product.yaml` per your other comment.